docs/fast_data/tutorials/low-code.mdx

```diff
@@ -484,51 +465,101 @@ It is important to also define the JSON Schema for the _order_ field of the Sing
 
-Now we will choose the System of Record and the Projection that we want to link to the Strategy; in our case, "delivery" and "registry".
-![Strategy Page](img/29-strategy-select-projection.png)
+![Marketplace and Single View Creator](img/23-marketplace-single-view-low-code.png)
 
-Last but not least, we need to choose the Strategy Source. In this case, we will use the "Low Code" strategy, then we can click on the **Create** button.
-![Strategy Page](img/30-strategy-low-code.png)
+Here we have two different plugins: the first one is for the Single View Creator, and the second one is for the Single View Creator Low Code.
+We will go for the latter since we want to leverage the usage of ConfigMaps for a quicker configuration of the microservice. We suggest naming it `single-view-creator-delivery`.
 
-Then we can create the strategy also to the Order Projection, following the same process.
-![Final Strategy Page](img/31-complete-strategies.png)
+Now we need to configure all the parameters of the microservice, here is a set of the fields that we need to configure:
+
+- `SINGLE_VIEWS_COLLECTION`: should be the name of the single view which your single view creator is responsible for (we suggest `order-history`, the same name as the Single View we are going to create);
+- `SINGLE_VIEWS_PORTFOLIO_ORIGIN`: a representative name used in debugging and metrics to identify the source of any Single View update request (we suggest to use the name of the System of Record, `delivery`);
+- `TYPE`: identifies the type of projection changes that need to be read. It should be the same as the Single View name you want to update (also here we are going to call it `order-history`).
+
+Also, based on the connection channel where the microservice reads the projection changes, additional environment variables needs to be set:
+
+<Tabs
+  defaultValue="kafka"
+  groupId="svc-connection"
+  values={[
+      { label: 'Kafka', value: "kafka", },
+      { label: 'MongoDB', value: "mongodb", },
+  ]}
+>
+
+<TabItem value="kafka">
+
+* `PROJECTION_CHANGES_SOURCE`: by using the value `kafka`, it instructs the service to read projection changes from a Kafka connection;
+* kafka connection variables such as:
+  * `KAFKA_BROKERS`: a comma-separated list of your Kafka brokers;
+  * `KAFKA_SASL_USERNAME`, `KAFKA_SASL_PASSWORD` and `KAFKA_SASL_MECHANISM` if authentication needs to be performed
+  * `KAFKA_PROJECTION_CHANGES_TOPIC`: name of the topic that will receive projection changes in the shape of [trigger messages](/fast_data/inputs_and_outputs.md#single-view-trigger-message), you can choose `demo.development.delivery.order-history.trigger`;
+  * `KAFKA_GROUP_ID`: name of the consumer group that will be used to subscribe to the topic from the `KAFKA_PROJECTION_CHANGES_TOPIC` variable.
```

Comment:
where is defined this KAFKA_GROUP_ID?

Reply:
@albertotessarotto it can be defined by the user, I added a suggestion